GCEWindowsAgent/wsfc.go

```diff
@@ -48,15 +50,31 @@ type wsfcManager struct {
 // agent request state will be set to running if one of the following is true:
 // - EnableWSFC is set
 // - WSFCAddresses is set (As an advanced setting, it will always override EnableWSFC flag)
```

Comment:
Please add some comment that WSFC = Windows Server Failover Clustering so people who don't know about it can more easily understand the guest code.